.github/workflows/reusable-misspell-check.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
       - name: Install misspell
         run: |
-          curl -L -o install-misspell.sh https://git.io/misspell
```

Comment:
https://git.io/misspell seems to be working again

Reply:
git.io is now locked into read-only mode, so it will never get updated, and I think it's going to be shut down very soon : https://github.blog/changelog/2022-04-25-git-io-deprecation/

Reply:
Oh dang thanks for that insight